src/component/accordion/Accordion.tsx

```diff
@@ -30,9 +35,8 @@ export const Accordion: FC = () => {
     <div className="accordion-wrapper">
       <div className="container">
         <div id="accordions" className="accordions">
-          {accordionList.map(({ question, answer }) => (
-            // eslint-disable-next-line react/jsx-key
-            <AccordionItems question={question} answer={answer} />
+          {accordionList.map(({ id, question, answer }) => (
+            <AccordionItems key={id} question={question} answer={answer} />
```

Comment:
since it is going to be a generic component Instead of question and answer please use some generic name. Like label & content or something more appropriate.